Added 3 new questions to infrastructure details UCR.

emord opened the pull request:
Will be a pretty quick build. only 200k forms

emord committed:
Added 3 new questions to infrastructure details UCR.

proteusvacuum commented:
```======================================================================
ERROR: test suite for <module 'custom.icds_reports.tests' from '/mnt/commcare-hq-ro/custom/icds_reports/tests/__init__.pyc'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 292, in setUp
    self.setupContext(ancestor)
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/vendor/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/__init__.py", line 100, in setUpModule
    postgres_copy.copy_from(f, table, engine, format=b'csv', null=b'', header=True)
  File "/vendor/lib/python2.7/site-packages/postgres_copy/__init__.py", line 78, in copy_from
    cursor.copy_expert(copy, source)
DataError: missing data for column "electricity_awc"
CONTEXT:  COPY config_report_icds-cas_static-infrastructure_form_05fe0f1a, line 2: "276de98b-6768-426b-92c2-f1158294cb52,2017-07-01,a12,s4,b2,d1,st1,2016-12-22,2016-12-01,,,,1,0,1,1,1,..."```

emord committed:
Added extra 3 columns to infrastructure form UCR.

This CSV is only used for the aggregation script and these columns are
not currently used in the aggregation script, so leaving them null for
now.
